### ISSUE TITLE
[Proposal] Recaps (Yearly, monthly, weekly, mood, stats)

### ISSUE THREAD

**Issue by Demizo** [Demizo/Daily_You#56]
Right how the home screen features a variety of flashbacks in the form of specific days. It would be nice to have auto-generated memory recaps that are collections of days and/or stats. 

There could also be a larger yearly recap similar to the Spotify recap. Otherwise, the recap could also change daily as it does now. Making some recaps only update after a set amount of time may make them more rewarding to view.

The current flashbacks would be trimmed down and new sections would be added (i.e. last week, last month, last year). Each section would have a collage of days (a happy highlight, a sad moment, etc.) the section could also have rudimentary stats (mood, engagement, etc.). These sections would update periodically.

**Comment by blaugrau3001** [comment]
Very good idea, but this will cost a lot of efforts when you want that not every slide looks the same.
Maybe you could get there with small steps for example first take a look at the issues #38 to make the data more visible and also issue #17 to get more photos to use in the slides.

**Comment by Demizo** [comment]
These are good points. Multiple photos per day should happen first to allow for collages. As it stands recaps could take the form of improved flashbacks. In other words, the could be embedded into the home screen in the same way flashbacks are now, but be boxed into sections (i.e. last week, last month, last year).

**Comment by blaugrau3001** [comment]
> These are good points. Multiple photos per day should happen first to allow for collages. As it stands recaps could take the form of improved flashbacks. In other words, the could be embedded into the home screen in the same way flashbacks are now, but be boxed into sections (i.e. last week, last month, last year).

I thought you mean it like google photos recap, if you know it

**Comment by Demizo** [comment]
> I thought you mean it like google photos recap, if you know it

I'm not familiar unfortunately. I do think a slideshow style recap would be great as well if that's what it is.

**Comment by Rafee-M** [comment]
> These are good points. Multiple photos per day should happen first to allow for collages. As it stands recaps could take the form of improved flashbacks. In other words, the could be embedded into the home screen in the same way flashbacks are now, but be boxed into sections (i.e. last week, last month, last year).


It would basically show a slideshow of happy days and sad days (could be in one or separate slideshows). It would basically be a slideshow of the last week, month or year
